aspnetcore/log-mon/metrics.md

```diff
@@ -0,0 +1,122 @@
+
+If the [dotnet-counters](/dotnet/core/diagnostics/dotnet-counters) tool isn't installed, run the following command:
+
+```dotnetcli
+dotnet tool update -g dotnet-counters
+```
+
+While the test app is running, launch [dotnet-counters](/dotnet/core/diagnostics/dotnet-counters). The following command shows an example of `dotnet-counters` monitoring all metrics from the `HatCo.HatStore` meter. The meter name is case-sensitive. The sample app is WebMetric.exe, change the name if you're using a different app name.
+
+```dotnetcli
+dotnet-counters monitor -n WebMetric
+```
+
+Output similar to the following is displayed:
+
+```dotnetcli
+Press p to pause, r to resume, q to quit.
+    Status: Running
+
+[System.Runtime]
+    % Time in GC since last GC (%)                                         0
+    Allocation Rate (B / 1 sec)                                        8,200
+    CPU Usage (%)                                                          0
+    Exception Count (Count / 1 sec)                                        0
+    GC Committed Bytes (MB)                                                0
+    GC Fragmentation (%)                                                   0
+    GC Heap Size (MB)                                                      8.823
+    Gen 0 GC Count (Count / 1 sec)                                         0
+    Gen 0 Size (B)                                                         0
+    Gen 1 GC Count (Count / 1 sec)                                         0
+    Gen 1 Size (B)                                                         0
+    Gen 2 GC Count (Count / 1 sec)                                         0
+    Gen 2 Size (B)                                                         0
+    IL Bytes Jitted (B)                                              530,289
+    LOH Size (B)                                                           0
+    Monitor Lock Contention Count (Count / 1 sec)                          0
+    Number of Active Timers                                                1
+    Number of Assemblies Loaded                                          120
+    Number of Methods Jitted                                           5,270
+    POH (Pinned Object Heap) Size (B)                                      0
+    ThreadPool Completed Work Item Count (Count / 1 sec)                   0
+    ThreadPool Queue Length                                                0
+    ThreadPool Thread Count                                                2
+    Time spent in JIT (ms / 1 sec)                                         0
+    Working Set (MB)                                                      78.852
+```
+
+The following command shows an example of `dotnet-counters` monitoring all metrics from the `System.Net.Http` meter.
+
+```dotnetcli
+dotnet-counters monitor -n WebMetric System.Net.Http:requests-started
+```
```

Comment:
@JamesNK  how do I get ASP.NET Core counters from `dotnet-counters list` ?

Microsoft.AspNetCore.Hosting
    requests-per-second                  Number of requests between update intervals
    total-requests                       Total number of requests
    current-requests                     Current number of requests
    failed-requests                      Failed number of requests

Microsoft-AspNetCore-Server-Kestrel
    connections-per-second               Number of connections between update intervals
    total-connections                    Total Connections
    tls-handshakes-per-second            Number of TLS Handshakes made between update intervals
    total-tls-handshakes                 Total number of TLS handshakes made
    current-tls-handshakes               Number of currently active TLS handshakes
    failed-tls-handshakes                Total number of failed TLS handshakes
    current-connections                  Number of current connections
    connection-queue-length              Length of Kestrel Connection Queue
    request-queue-length                 Length total HTTP request queue

System.Net.Http
    requests-started                             Total Requests Started
    requests-started-rate                        Number of Requests Started between update intervals
    requests-aborted                             Total Requests Aborted
    requests-aborted-rate                        Number of Requests Aborted between update intervals
    current-requests                             Current Requests
    http11-connections-current-total             Current number of HTTP 1.1 connections
    http20-connections-current-total             Current number of HTTP 2.0 connections
    http11-requests-queue-duration               Average duration of the time HTTP 1.1 requests spent in the request queue
    http20-requests-queue-duration               Average duration of the time HTTP 2.0 requests spent in the request queue

Reply:
I don't know. I've used the tool a little and don't know enough to say what is wrong.